ui/ui_vfinal/intents/javascript_functions/submit_query.html

```diff
@@ -219,11 +288,19 @@
       
     if (suggestionsContent.style.maxHeight){
       suggestionsContent.style.maxHeight = null;
+      hideNextButton();
+      hidePreviousButton();
+      index=0;
     } 
     else {
       suggestionsContent.style.maxHeight = suggestionsContent.scrollHeight + 'px';
+      // index check rqd
+      index=0;
```

Comment:
We are resetting the index to 0, but not calling showSuggestions(index) again to show the correct suggestion.

Reply:
Corrected. we only will only call `showSuggestions(index)` only when the suggestion list is not empty.